lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -141,6 +145,19 @@
   }
 }
 
+.nx-dropdown-link .nx-icon {
+  &:hover {
+    color: var(--nx-color-link-hover);
```

Comment:
Is this needed? Even without it I'm seeing the link icons change color when the link is hovered

Reply:
> Is this needed? Even without it I'm seeing the link icons change color when the link is hovered

It's for the close icon next to the reset button under the filter dropdown.

Reply:
> Is this needed? Even without it I'm seeing the link icons change color when the link is hovered

Yeah, it isn't needed anymore: 602eb123d4617a696be4b955b4cdb781fd26ed1b